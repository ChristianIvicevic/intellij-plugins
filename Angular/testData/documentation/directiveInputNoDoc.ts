import {Component, Directive, Input} from "@angular/core";

@Directive({
    standalone: true,
    selector: "[my-input]"
})
export class MyDirective {
    @Input("my-input")
    myInput: String
}

@Component({
    standalone: true,
    template: "<div my-<caret>input=''></div>",
    imports: [
        MyDirective
    ]
})
class MyComponent {
}
