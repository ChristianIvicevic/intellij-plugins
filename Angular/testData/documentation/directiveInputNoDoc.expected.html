<div class='definition'><pre><span style='color: #000080'>directive input</span> <span style='color: #660e7a'>my-input</span><span style='color: #000000'>: </span><span style='color: #000000'><span style=""><span style='color: #589df6'><a href="psi_element://String">String</a></span></span></span></pre></div>
<div class='definition'><pre><span style='color: #000080'>standalone</span> <span style='color: #000080'>directive</span> <span style='color: #000000'><span style="">MyDirective</span></span></pre></div><table class='sections'><tr><td valign='top' class='section'><p>Selectors:</td><td valign='top'><p><pre><span style="">[</span><span style="color:#000080;">my-input</span><span style="">]</span></pre></td></table>